### PR TITLE
Added hbss

### PR DESCRIPTION
**Pull request by yahyaHB**
Added the following files:

- navbar.hbs
- footer.hbs
- style sheet of nav, footer files

### CODE REVIEW

**Review by NouraldinS — CHANGES_REQUESTED** [review]
remove the footer and header tags, I already included those in the main.hbs

**Comment by sultanassi95** [comment]
@NoureldeanSaed I think there is no need to do that as he is using div elements with classes 'navbar, footer) , which are not the same as navbar and footer elements u have in pull request.
maybe we should wait until pull request #41 is merged first and then discuss this one.

**Review by sultanassi95 — COMMENTED** [review]
holding until #41 is merged.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "express-handlebars": "^3.0.0",
     "path": "^0.12.7",
     "pg": "^7.4.1",
-    "serve-favicon": "^2.4.5",
+    "serve-favicon": "^2.4.5"
```
Comment (mohanagy) [inline package.json:15]: i don't see that u using this module in your project ?

File: postProject.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<html>
+
+<head>
+  <link rel="stylesheet" href="postProject.css">
+  <meta charset="utf-8">
+  <title>Post a project</title>
+</head>
+
+<body>
+  <!-- <section>
```
Comment (mohanagy) [inline postProject.html]: remove commented lines

**Review by mohanagy — CHANGES_REQUESTED** [review]

File: postProject.html (hunk 2 above)
Comment (yahyaHB) [reply postProject.html]: i'm working on the content :+1:

Comment (mohanagy) [reply postProject.html]: ok remove it :)

**Review by NouraldinS — APPROVED** [review]

**Review by sultanassi95 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mohanagy) [reply package.json:15]: ?

Comment (NouraldinS) [reply package.json:15]: Check the other pull requests, especially the ones been waiting for more than 3 hours...

Comment (NouraldinS) [reply package.json:15]: @mohanagy

**Pull request closed without merging**